eacl/target.go

```diff
@@ -58,6 +60,27 @@ func (t *Target) SetBinaryKeys(keys [][]byte) {
 	t.keys = keys
 }
 
+// Accounts returns list of accounts to identify target subject in a binary format.
+//
+// Each element of the resulting slice is a 25 byte account.
+// Use `user := user.ID(slice)` to decode it into a type-specific structure.
+//
+// Instead of an account there can also be a 33 byte public key binary.
+// See [keys.PublicKey].
+//
+// The value returned shares memory with the structure itself, so changing it can lead to data corruption.
+// Make a copy if you need to change it.
+func (t *Target) Accounts() [][]byte {
```

Comment:
lets deprecate `BinaryKeys` and `SetBinaryKeys` then

Reply:
Don't we plan to use pub keys later?

Reply:
they are already supported by `SetAccounts`

Reply:
Done